datastore/gcloud/aio/datastore/value.py

```diff
@@ -82,5 +85,6 @@ def _infer_type(self, value: Any) -> TypeName:
     def _get_supported_types(cls):
         return {
             **TYPES,
+            LatLng: TypeName.GEOPOINT,
```

Comment:
This is already in `TYPES`, no need to re-add it here

Reply:
Aha, believe it or not, I have this change locally but it didn't make it into the commit! Amending now